File: Individual-Biblioteca/Backend-biblioteca/backend-biblioteca/api/models/Libro.js
/**
 * Libro.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

    titulo: {
      type: 'string',
      required: true
    },

    autor: {
      type: 'string',
      required: true
    },

    editorial: {
      type: 'string',
      required: true
    },

    numeroEdicion: {
      type: 'string',
      required: true
    },

    stock: {
      type: 'number',
      required: true
    },

    LibroSolicitud: {
      collection: 'solicitud',
      via: 'idLibro'
    }

  },

};

File: Proyecto_Individual_Biblioteca/app/src/main/java/com/example/proyecto_biblioteca/Libros/Libro.kt
package com.example.proyecto_biblioteca.Libros

class Libro (var createdAt: Long?,
             var updatedAt: Long?,
             var id: Int?,
             var titulo: String,
             var autor: String,
             var numeroEdicion: String,
             var editorial: String,
             var stock: Int){
}